repository-data/application/src/main/resources/hcm-config/namespaces/hee/MiniHub.yaml

```diff
@@ -0,0 +1,100 @@
+definitions:
+  config:
+    /hippo:namespaces/hee/MiniHub:
+      jcr:primaryType: hipposysedit:templatetype
+      jcr:mixinTypes: ['editor:editable', 'mix:referenceable']
+      jcr:uuid: 1d6b162b-5451-4bf3-abac-bc8dc868b5cd
+      /hipposysedit:nodetype:
+        jcr:primaryType: hippo:handle
+        jcr:mixinTypes: ['mix:referenceable']
+        jcr:uuid: 3fed70ec-498a-4b0d-8537-c2b4c12b3a0d
+        /hipposysedit:nodetype:
+          jcr:primaryType: hipposysedit:nodetype
+          jcr:mixinTypes: ['mix:referenceable', 'hipposysedit:remodel']
+          jcr:uuid: 95f0d1c7-2675-4e8c-b694-24bf4c997d05
+          hipposysedit:node: true
+          hipposysedit:supertype: ['hee:basedocument', 'hippostd:relaxed', 'hippotranslation:translated']
+          hipposysedit:uri: http://www.heecmsplatform.com/hee/nt/1.0
+          /title:
+            jcr:primaryType: hipposysedit:field
+            hipposysedit:mandatory: false
+            hipposysedit:multiple: false
+            hipposysedit:ordered: false
+            hipposysedit:path: hee:title
+            hipposysedit:primary: false
+            hipposysedit:type: String
+            hipposysedit:validators: [required]
+          /summary:
+            jcr:primaryType: hipposysedit:field
+            hipposysedit:mandatory: false
+            hipposysedit:multiple: false
+            hipposysedit:ordered: false
+            hipposysedit:path: hee:summary
+            hipposysedit:primary: false
+            hipposysedit:type: String
+            hipposysedit:validators: [required]
+          /guidances:
```

Comment:
@thojava - Could we make `guidances` field mandatory as I guess every minihub page should at least have one or more guidances ? (Otherwise, I think the page isn't really a candidate for minihub)

Reply:
Would this make more sense named `Guidance pages`